Resources/doc/index.rst

```diff
@@ -148,53 +156,24 @@ like the following:
         }
     }
 
-If you run the ``status`` command it will now show that you have one new
+If you run again the ``status`` command it will now show that you have one new
 migration to execute:
 
 .. code-block:: terminal
 
     $ php bin/console doctrine:migrations:status --show-versions
 
-     == Configuration
-
-        >> Name:                                               Application Migrations
-        >> Database Driver:                                    pdo_mysql
-        >> Database Host:                                      127.0.0.1
-        >> Database Name:                                      symfony_migrations
-        >> Configuration Source:                               manually configured
-        >> Version Table Name:                                 migration_versions
-        >> Version Column Name:                                version
-        >> Migrations Namespace:                               DoctrineMigrations
-        >> Migrations Directory:                               /path/to/project/app/Migrations
-        >> Previous Version:                                   Already at first version
-        >> Current Version:                                    0
-        >> Next Version:                                       2018-06-05 02:56:53 (20180605025653)
-        >> Latest Version:                                     2018-06-05 02:56:53 (20180605025653)
-        >> Executed Migrations:                                0
-        >> Executed Unavailable Migrations:                    0
-        >> Available Migrations:                               1
-        >> New Migrations:                                     1
-
-     == Available Migration Versions
-
-        >> 2018-06-05 02:56:53 (20180605025653)                not migrated
-
 Now you can add some migration code to the ``up()`` and ``down()`` methods and
 finally migrate when you're ready:
 
 .. code-block:: terminal
 
-    $ php bin/console doctrine:migrations:migrate 20180605025653
+    $ php bin/console doctrine:migrations:migrate 'DoctrineMigrations\Version20180605025653'
```

Comment:
Seeing this makes me realise how important it is that we show examples, since people may not be aware that they need to escape backslashes in shell arguments.

We may want to add a comment to highlight this and to tell people not to use double quotes (`"`) to quote the class names.